frameworks/desktop/views/scroller.js

```diff
@@ -599,6 +599,21 @@ SC.ScrollerView = SC.View.extend(
     return value;
   },
 
+  // [CC] scroll bar supports touch events
+  touchStart: function (evt) {
+    evt.which = 1;
```

Comment:
I don't see 'which' as a property of a touch event here: https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent. What is the significance of setting it to 1?

Reply:
We're passing the event to the `mouseDown()` handler below which only processes it if `evt.which` is 1, so we're making it look just enough like a mouse event that the `mouseDown()` handler won't reject it.